suite-native/transactions/src/components/TransactionDetail/TransactionDetailHeader.tsx

```diff
@@ -39,6 +44,9 @@ export const TransactionDetailHeader = ({ type, amount }: TransactionDetailHeade
                 <Text style={applyStyle(transactionTypeStyle)}>{transactionTypeTextMap[type]}</Text>
             </Box>
             <Text variant="titleMedium">{amount}</Text>
+            <Text variant="label" color="gray700">
+                ≈ {fiatAmount}
```

Comment:
```suggestion
                ≈ {fiatAmount}
```
<FiatAmountFormatter currency={targetCurrency} value={fiatAmount} />

I would likely do it this way. Pass pure `fiatAmount` (you are currently passing formatted fiat amount by `FiatAmountFormatter.format` and call it fiatAmount) to this component. Then select your local currency with:

`const fiatCurrency = useSelector(selectFiatCurrency);`

Format it as suggested.

Reply:
[0b65f2d](https://github.com/trezor/trezor-suite/pull/6568/commits/0b65f2d08c3530a7ca8638c400363bd77222c07c)